runbot_buildout/static/src/js/runbot_buildout.js

```diff
@@ -0,0 +1,16 @@
+//-*- coding: utf-8 -*-
```

Comment:
Minor change, but do we actually need this?

Reply:
not at all, thanks. That's a fluke from my addons template, fixed it in the template and will remove this on next push